src/autogluon/cloud/predictor/timeseries_cloud_predictor.py

```diff
@@ -0,0 +1,268 @@
+            Number of instance used to fit the predictor.
+        volumes_size: int, default = 30
+            Size in GB of the EBS volume to use for storing input data during training (default: 30).
+            Must be large enough to store training data if File Mode is used (which is the default).
+        wait: bool, default = True
+            Whether the call should wait until the job completes
+            To be noticed, the function won't return immediately because there are some preparations needed prior fit.
+            Use `get_fit_job_status` to get job status.
+        autogluon_sagemaker_estimator_kwargs: dict, default = dict()
+            Any extra arguments needed to initialize AutoGluonSagemakerEstimator
+            Please refer to https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html#sagemaker.estimator.Framework for all options
+        **kwargs:
+            Any extra arguments needed to pass to fit.
+            Please refer to https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html#sagemaker.estimator.Framework.fit for all options
+
+        Returns
+        -------
+        `TimeSeriesCloudPredictor` object. Returns self.
+        """
+        predictor_fit_args = copy.deepcopy(predictor_fit_args)
+        train_data = predictor_fit_args.pop("train_data")
+        tuning_data = predictor_fit_args.pop("tuning_data", None)
+        target = predictor_init_args.get("target")
+        train_data = self._preprocess_data(
+            data=train_data,
+            id_column=id_column,
+            timestamp_column=timestamp_column,
+            target=target,
+            static_features=static_features,
+        )
+        if tuning_data is not None:
+            tuning_data = self._preprocess_data(
+                data=tuning_data,
+                id_column=id_column,
+                timestamp_column=timestamp_column,
+                target=target,
+                static_features=static_features,
+            )
+        predictor_fit_args["train_data"] = train_data
+        predictor_fit_args["tuning_data"] = tuning_data
+        print(train_data)
+        return super().fit(
+            predictor_init_args=predictor_init_args,
+            predictor_fit_args=predictor_fit_args,
+            framework_version=framework_version,
+            job_name=job_name,
+            instance_type=instance_type,
+            instance_count=instance_count,
+            volume_size=volume_size,
+            custom_image_uri=custom_image_uri,
+            wait=wait,
+            autogluon_sagemaker_estimator_kwargs=autogluon_sagemaker_estimator_kwargs,
+            **kwargs,
+        )
+
+    def predict_real_time(
+        self,
+        test_data: Union[str, pd.DataFrame],
+        id_column: str,
+        timestamp_column: str,
+        target: str,
+        static_features: Optional[Union[str, pd.DataFrame]] = None,
+        accept: str = "application/x-parquet",
+    ) -> pd.DataFrame:
```

Comment:
This is returning a pd.Series

Reply:
This is actually a bug. Should be fixed now.